e2e-with-targets/01_creating_database.R

```diff
@@ -0,0 +1,38 @@
+## database.R
+
+# loading libraries -------------------------------------------------------------------------------
+box::use(
+  dplyr[mutate, relocate, where, select],
+  RSQLite[SQLite],
+  DBI[dbConnect, dbWriteTable, dbDisconnect]
+)
+
+# preparing data sets for database ----------------------------------------------------------------
+mtcars <- datasets::mtcars
+iris <- datasets::iris
+
+mtcars_df_proper <- mtcars |>
+  mutate(name = rownames(mtcars)) |>
+  `rownames<-`(NULL) |>
+  mutate(id = 1:nrow(mtcars)) |>
```

Comment:
I suggest using `row_number` or `tibble::rowid_to_column()` - it looks cleaner
```suggestion
  mutate(id = dplyr::row_number()) |>
```